apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -407,6 +425,8 @@ public void startTestUsingBundle() {
             // Specific options.
             mShouldUseADPF = mBundleFromIntent.getBoolean(KEY_USE_ADPF,
                     VALUE_DEFAULT_USE_ADPF);
+            mShouldUseWorkloadReporting = mBundleFromIntent.getBoolean(KEY_USE_WORKLOAD,
```

Comment:
Please update https://github.com/google/oboe/blob/main/apps/OboeTester/docs/AutomatedTesting.md with the new parameter.